dpnp/dpnp_iface_manipulation.py

```diff
@@ -3894,18 +3894,13 @@ def transpose(a, axes=None):
 
     """
 
-    if isinstance(a, dpnp_array):
-        array = a
-    elif isinstance(a, dpt.usm_ndarray):
-        array = dpnp_array._create_from_usm_ndarray(a)
-    else:
-        raise TypeError(
-            f"An array must be any of supported type, but got {type(a)}"
-        )
+    dpnp.check_supported_arrays_type(a)
```

Comment:
Could you please update the description of `axis` keyword and to state default value for it?